Source/DedicatedServer/Private/ServerConsoleWindows.cpp

```diff
@@ -45,31 +47,42 @@ void DumpConsoleHelp();
 					ClearInputLine();
 
 					m_pConsole->SetColor( COLOR_GREEN );
-
-					TCHAR sOutput[MAX_SPRINTF] = TEXT( "" );
-					unsigned long iCharsWritten;
-
-					if( m_sInput.StartsWith( TEXT( "help" ) ) )
-					{
-						DumpConsoleHelp();
-
-						FCString::Sprintf( sOutput, TEXT( "> %s%s" ), *m_sInput, LINE_TERMINATOR );
-						::WriteConsole( m_hOutputHandle, sOutput, FCString::Strlen( sOutput ), &iCharsWritten, NULL );
-
-						m_hCommandHistory.Add( m_sInput );
-					}
-					else if( GEngine->Exec( GEngine->GetWorld(), *m_sInput ) )
-					{
-						FCString::Sprintf( sOutput, TEXT( "> %s%s" ), *m_sInput, LINE_TERMINATOR );
-						::WriteConsole( m_hOutputHandle, sOutput, FCString::Strlen( sOutput ), &iCharsWritten, NULL );
-
-						m_hCommandHistory.Add( m_sInput );
-					}
-					else
-					{
-						FCString::Sprintf( sOutput, TEXT( "Unknown Command: %s%s" ), *m_sInput, LINE_TERMINATOR );
-						::WriteConsole( m_hOutputHandle, sOutput, FCString::Strlen( sOutput ), &iCharsWritten, NULL );
-					}
+					auto& tmp = m_sInput;
+					AsyncTask(ENamedThreads::GameThread, [this, tmp]() {
+						// code to execute on game thread here
+
+						TCHAR sOutput[MAX_SPRINTF] = TEXT("");
+						unsigned long iCharsWritten;
```

Comment:
```suggestion
						unsigned long iCharsWritten = 0;
```
It's not really used, but I guess initializing it isn't such a bad idea anyway (not sure, but maybe `&iCharsWritten` could be changed to `nullptr` or simply 0 in that case, I don't think that crossed my mind back then...)